packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -19,6 +19,7 @@ type TDetailedExistingAccount = AvailableAccount &
     TDetailsOfEachMT5Loginid &
     Actions & {
         key: string;
+        webtrader_url: string;
```

Comment:
We dont need to add this as this should be coming from backend in `DetailsOfEachMT5Loginid` which is extended in `TDetailedExistingAccount`.
It should be in `deriv/api-types` maybe we need to update `deriv/api-types`.
```suggestion
        
```
<img width="1324" alt="Screenshot 2023-09-21 at 12 21 27 PM" src="https://github.com/binary-com/deriv-app/assets/120543468/c824fc0b-e1ff-4f4c-8197-abf9261db137">